crates/oxc_transformer/src/react/refresh.rs

```diff
@@ -985,3 +986,13 @@ fn get_symbol_id_from_function_and_declarator(stmt: &Statement<'_>) -> Vec<Symbo
 
     symbol_ids
 }
+
+#[test]
+fn test_is_hook_name() {
```

Comment:
A integration test is preferred, add to an existing test case?

Reply:
Done